Source/WebCore/platform/graphics/filters/FilterOperations.h

```diff
@@ -96,6 +94,12 @@ class FilterOperations {
     Vector<Ref<FilterOperation>> m_operations;
 };
 
+template<FilterOperation::Type Type>
+bool FilterOperations::hasFilterOfType() const
```

Comment:
I typically prefer the one-line format for these, with `template` on the same line as the function signature.

Since the argument is a non-class type, can it be lowercase `type` instead of uppercase `Type`?